### ISSUE TITLE
local variable 'custom_css_empty' referenced before assignment at start notebook in Ubuntu14

### ISSUE THREAD

**Issue by Overvenus** [jupyter/jupyter_core#58]
I installed ipython several monthes ago, so I installed jupyter following the [installation, Existing IPython Notebook users upgrading to Jupyter Notebook](https://jupyter.readthedocs.org/en/latest/install.html#existing-ipython-notebook-users-upgrading-to-jupyter-notebook)

I had this error when I started notebook. It looks like #52 ,but actually it not.
```python
$ sudo pip install -U jupyter
...
...
$ jupyter notebook

Traceback (most recent call last):
  File "/usr/local/bin/jupyter-notebook", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/jupyter_core/application.py", line 267, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/traitlets/config/application.py", line 591, in launch_instance
    app.initialize(argv) File "<string>", line 2, in initialize
  File "/usr/local/lib/python2.7/dist-packages/traitlets/config/application.py", line 75, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/notebook/notebookapp.py", line 1001, in initialize
    super(NotebookApp, self).initialize(argv) File "<string>", line 2, in initialize
  File "/usr/local/lib/python2.7/dist-packages/traitlets/config/application.py", line 75, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/jupyter_core/application.py", line 243, in initialize
    self.migrate_config()
  File "/usr/local/lib/python2.7/dist-packages/jupyter_core/application.py", line 169, in migrate_config
    migrate() 
  File "/usr/local/lib/python2.7/dist-packages/jupyter_core/migrate.py", line 232, in migrate
    if migrate_static_custom(custom_src, custom_dst): 
  File "/usr/local/lib/python2.7/dist-packages/jupyter_core/migrate.py", line 160, in migrate_static_custom
    if custom_css_empty:
UnboundLocalError: local variable 'custom_css_empty' referenced before assignment
```

Here is my `~/.ipython` directory:
```
$ tree -a ~/.ipython                                                         
/home/neil/.ipython
├── [drwxrwxr-x]  extensions
├── [drwxrwxr-x]  nbextensions
├── [drwxrwxr-x]  profile_default
│   ├── [drwxrwxr-x]  db
│   ├── [-rw-r--r--]  history.sqlite
│   ├── [drwxrwxr-x]  log
│   ├── [drwx------]  pid
│   ├── [drwx------]  security
│   ├── [drwxrwxr-x]  startup
│   │   └── [-rw-r--r--]  README
│   └── [drwxrwxr-x]  static
│       └── [drwxrwxr-x]  custom
└── [-rw-r--r--]  README

10 directories, 3 files
```

This error can be reproduced in my computer using the scripts `./scripts/jupyter-migrate` 

```python
$ ./scripts/jupyter-migrate 

Traceback (most recent call last):
  File "./scripts/jupyter-migrate", line 7, in <module>
    main()
  File "/usr/local/lib/python2.7/dist-packages/jupyter_core/application.py", line 267, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/traitlets/config/application.py", line 592, in launch_instance
    app.start()
  File "/usr/local/lib/python2.7/dist-packages/jupyter_core/migrate.py", line 263, in start
    if not migrate():
  File "/usr/local/lib/python2.7/dist-packages/jupyter_core/migrate.py", line 232, in migrate
    if migrate_static_custom(custom_src, custom_dst):
  File "/usr/local/lib/python2.7/dist-packages/jupyter_core/migrate.py", line 160, in migrate_static_custom
    if custom_css_empty:
UnboundLocalError: local variable 'custom_css_empty' referenced before assignment
```

My system information:
```sh
$ python -c "import IPython; print(IPython.sys_info())"
{'commit_hash': u'f534027',
 'commit_source': 'installation',
 'default_encoding': 'UTF-8',
 'ipython_path': '/usr/local/lib/python2.7/dist-packages/IPython',
 'ipython_version': '4.0.0',
 'os_name': 'posix',
 'platform': 'Linux-3.13.0-62-generic-x86_64-with-Ubuntu-14.04-trusty',
 'sys_executable': '/usr/bin/python',
 'sys_platform': 'linux2',
 'sys_version': '2.7.6 (default, Jun 22 2015, 17:58:13) \n[GCC 4.8.2]'}

```

**Comment by minrk** [comment]
@Overvenus sorry about that. Fix merged, released as 4.0.6.

**Comment by Overvenus** [comment]
Thanks!
